templates/cluster_name.conf.erb

```diff
@@ -13,3 +13,5 @@ api_port = <%= @cassandra_api_port %>
 
 [storage_cassandra]
 seed_hosts = <%=  @cassandra_seed_hosts %>
+username = <%= @cassandra_username %>
```

Comment:
Good for scope, but we should have storage_cassandra_username and storage_cassandra_password to allow the storage to be completely decoupled. Ideal scenario is that OpsCenter(s) use a central cluster for storage. That way also the datastax-agent can communicate on clean (non-SSL) channel with its storage.